ssl/quic/quic_channel.c

```diff
@@ -2721,8 +2781,23 @@ static void ch_record_state_transition(QUIC_CHANNEL *ch, uint32_t new_state)
                                                           ch->handshake_confirmed);
 }
 
+static void free_token(const unsigned char *token, size_t token_len, void *arg)
```

Comment:
Why is this moved??

Reply:
Inadvertently done.  I had initially intended to use free_token for multiple use cases, but the implementation made it not particularly beneficial.  I can move it back